samples/Azure.Management.Storage/Generated/BlobContainersCreateOrUpdateImmutabilityPolicyOperation.cs

```diff
@@ -11,30 +11,29 @@
 using Azure;
 using Azure.Core;
 using Azure.Management.Storage.Models;
-using Azure.ResourceManager.Core;
 
 namespace Azure.Management.Storage
 {
     /// <summary> Creates or updates an unlocked immutability policy. ETag in If-Match is honored if given but not required for this operation. </summary>
-    public partial class BlobContainersCreateOrUpdateImmutabilityPolicyOperation : Operation<BlobContainer>
+    public partial class BlobContainersCreateOrUpdateImmutabilityPolicyOperation : Operation<ImmutabilityPolicy>
```

Comment:
This is an example showing not ALL PUT API should wrap the result type in [Resource], so I narrow down the condition to also detect result type == `[Resource]Data`